client/cli/src/lib.rs

```diff
@@ -112,3 +118,135 @@ impl sc_cli::CliConfiguration for PurgeChainCmd {
 		Some(&self.base.database_params)
 	}
 }
+
+/// The `run` command used to run a node.
+#[derive(Debug, StructOpt)]
+pub struct RunCmd {
+	/// The cumulus RunCmd inherents from sc_cli's
+	#[structopt(flatten)]
+	pub base: sc_cli::RunCmd,
+
+	/// Id of the parachain this collator collates for.
+	#[structopt(long)]
+	pub parachain_id: Option<u32>,
+
+	/// Run node as collator.
+	///
+	/// Note that this is the same as running with `--validator`.
+	#[structopt(long, conflicts_with = "validator")]
+	pub collator: bool,
+}
+
+/// A non-redundant version of the `RunCmd` that sets the `validator` field when the
+/// original `RunCmd` had the `colaltor` field.
+///Basically this is how we make `--collator` imply `--validator`.
+pub struct NormalizedRunCmd {
+	/// The cumulus RunCmd inherents from sc_cli's
+	pub base: sc_cli::RunCmd,
+	/// Id of the parachain this collator collates for.
+	pub parachain_id: Option<u32>,
+}
+
+impl RunCmd {
+	/// Transform this RunCmd into a NormalizedRunCmd which does not have a redundant `collator` field.
+	pub fn normalize(&self) -> NormalizedRunCmd {
```

Comment:
```suggestion
	/// Transform this RunCmd into a NormalizedRunCmd which does not have a redundant `collator` field.
	pub fn normalize(&self) -> NormalizedRunCmd {
```

Do you maybe have a better idea for the function name?

And the docs should also be updated

Reply:
Do you not like `normalize`? I like it because it indicates that we aren't really changing any of the data, just writing it in a more standard form. `standardize` would also be fine. I like both of those better than the original `transform`.

I'm fine with any, of them, but I can't spend too long thinking about it because I have a bike shed I have to paint today ;-)

Reply:
I updated the docs as you requested.

Reply:
Fine